packages/@lwc/engine/src/__tests__/dom.spec.ts

```diff
@@ -18,8 +18,8 @@ describe('dom', () => {
             });
             elm.click();
         });
-        // Flapper
```

Comment:
I recently saw this in another PR and didn't understand how such a simple test could flap. Did we change something to address that?

Reply:
well, we are NOT longer using synthetic shadow in jest, will that help? I think it does.